--- /usr/bin/kernel-install	2022-11-08 20:38:19.000000000 +0100
+++ /usr/bin/kernel-install	2022-11-13 10:42:16.318948454 +0100
@@ -159,7 +159,7 @@
         log_verbose "machine-id $MACHINE_ID acquired from /etc/machine-info"
 fi
 if [ -z "$MACHINE_ID" ] && [ -f /etc/machine-id ]; then
-    read -r MACHINE_ID </etc/machine-id
+    read -r MACHINE_ID </etc/machine-id || true
     [ -n "$MACHINE_ID" ] && \
         log_verbose "machine-id $MACHINE_ID acquired from /etc/machine-id"
 fi
